Rakefile

```diff
@@ -1,6 +1,7 @@
 require "yast/rake"
 
 Yast::Tasks.submit_to :casp10
+obs_project = "Devel:YaST:CASP:1.0"
```

Comment:
What's the purpose of this line. The assignation seems to be going nowhere (`obs_project` is just a local variable) and the `submit_to` line just above is supposed to already target the proper OBS project.